docs/_pages/manuals.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Manuals"
+permalink: /manuals/
+layout: splash
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/amc_banner.jpg
+#   actions:
+#     - label: "Download"
+#       url: "https://github.com/mmistakes/minimal-mistakes/"
+  caption: "Photo credit: [**buguroo**](https://www.buguroo.com/en/blog/topic/ai)"
+excerpt: "AI ML Club"
+toc: true
+toc_label: "Unique Title"
+toc_icon: "heart"  # corresponding Font Awesome icon name (without fa prefix)
+---
+
+## AI ML Club Manuals
+
+- [Code of Conduct](/code-of-conduct/)
+- Manuals
+  - [Newcomer Manual](/manuals/newcomer/)
+  - [Member Manual](/manuals/member/)
+  - [Chair Manual](/manuals/chair/)
+  - [Vice Chair Manual](/manuals/vice-chair/)
+  - [Pull Request Manual](/manuals/pull-request/)
+  - [Blog Manual](/manuals/blog/)
+  - [Code Manual](/manuals/code/)
```

Comment:
Again, I believe we had already move this file to the manuals/ directory.

So this file should be created again.

Reply:
I have deleted duplicated files in my new commit.